### PR TITLE
Break up test acceptance-tests-large (--acceptance-go-client-named-vectors)

### LINKED ISSUE #5951: [Tech Debt] Break up test `acceptance-tests-large` (`--acceptance-go-client-named-vectors`)
The test mentioned in the title runs 3 or 4 independent test suites in sequence. This is enough to push it over the default 10m timeout and as a result the timeout has been increased. To make the CICD pipeline faster, it would be better if this test was split up into multiple smaller test jobs that can run in parallel.

### PR DESCRIPTION
**Pull request by melsonic**
### What's being changed:
I have split the `--acceptance-go-client-named-vectors` test into two independent tests to reduce the test running time and make CI/CD pipeline faster
- `--acceptance-go-client-named-vectors-single-node`
- `--acceptance-go-client-named-vectors-cluster`

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

fixes #5951

### CODE REVIEW

**Comment by moogacs** [comment]
@melsonic  thanks for opening the PR 😊 . I will add some feedback shortly

**Review by moogacs — COMMENTED** [review]
Looks good! have a comment see above. 

- could you follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for the PR title
something like
e.g.  `chore(acceptance-tests): break up test acceptance-tests-large (--acceptance-go-client-named-vectors)`
- also could you open the PR against `stable/v1.25` branch instead of `main`

**Comment by moogacs** [comment]
also could you open the PR against `stable/v1.25` branch instead of main

File: test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go
[the remark this answers is not in the document]
Comment (melsonic) [reply test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go:1]: I was facing some import issues with `named_vectors_tests`, so I renamed the package name. I will again give it a go to resolve those parameters.
I have one question, should I also rename the package name to `named_vectors_tests` or is it okay to keep dir name `named_vectors_tests` and package name `named_vectors`?

**Comment by melsonic** [comment]
> Looks good! have a comment see above.
> 
> * could you follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for the PR title
>   something like
>   e.g.  `chore(acceptance-tests): break up test acceptance-tests-large (--acceptance-go-client-named-vectors)`
> * also could you open the PR against `stable/v1.25` branch instead of `main`

yeah, I will make these changes.
thank you for your suggestions.

**Comment by melsonic** [comment]
> Looks good! have a comment see above.
> 
> * could you follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for the PR title
>   something like
>   e.g.  `chore(acceptance-tests): break up test acceptance-tests-large (--acceptance-go-client-named-vectors)`
> * also could you open the PR against `stable/v1.25` branch instead of `main`

yeah, I will make these changes.
thank you for your suggestions.

**Comment by melsonic** [comment]
Hey @moogacs, I had some problems while cherry-picking commits from the `melsonic` branch onto the branch created from `stable/v1.25`, hence created a new branch with a new `[PR](https://github.com/weaviate/weaviate/pull/6125)` for this issue. Please review

thank you

File: test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go
[the remark this answers is not in the document]
Comment (moogacs) [reply test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go:1]: would be better if the folder and package name match , and also to decrease diff

**Comment by moogacs** [comment]
@melsonic  thanks , i will close this PR as it's replaced by https://github.com/weaviate/weaviate/pull/6125 , feel free to reopen if you think otherwise 😊

**Pull request closed without merging**